src/compas/scene/sceneobject.py

```diff
@@ -52,7 +56,7 @@ def draw(self):
         """The main drawing method."""
         raise NotImplementedError
 
-    @staticmethod
-    def draw_collection(collection):
-        """Drawing method for drawing an entire collection of objects."""
-        raise NotImplementedError
+    def clear(self):
+        """The main clearing method."""
+        clear(guids=self.guids)
+        self._guids = None
```

Comment:
Then we also use same same `clear` function for individual object. In this way the clear function of the object doesn't need to be re-implemented anymore